### ISSUE TITLE
Remove epoch references for investors 

### ISSUE THREAD

**Issue by annamehr** [centrifuge/apps#1201]
## Background
After a few interviews and a discussion during the product offiste we have found some areas where we can make the app more comprehensible. Abstract epoch references away and make the flow of investment and redemption more comprehensible for a user who has not heard about epochs before.

### Liquidity tab
[Link to Figma ](https://www.figma.com/file/d1bEBkwllUQe27nQM6VlJu/Centrifuge-app---Version-1-(Master)?node-id=1971%3A53378&t=yVw9ntJ3z4p0WkFF-0) 

#### Changes for all status 
<img width="727" alt="Screenshot 2023-02-17 at 15 38 18" src="https://user-images.githubusercontent.com/31280003/219684666-d0884c7a-08e6-48b3-9a1a-4f65dadfc68d.png">

* Change `epoch n` to `Order overview`
* Change `An epoch is a period of locking investments and redemptions. [Learn more](https://docs.centrifuge.io/learn/epoch/)`
to `[Learn more](https://docs.centrifuge.io/learn/epoch/) about how orders are processed.`

#### Status Min. duration ended
<img width="516" alt="Screenshot 2023-02-17 at 15 43 20" src="https://user-images.githubusercontent.com/31280003/219686183-2a337fb1-7547-4f80-aa99-0e9e6addea73.png">

- Change button from `Closing epoch...` to `Closing...`

#### Status Closed
<img width="520" alt="Screenshot 2023-02-17 at 15 46 21" src="https://user-images.githubusercontent.com/31280003/219687719-2015d89b-6fc5-47e9-9d10-742c44c4819f.png">

- Change from [Countdown] to `Pending orders can be executed in [insert countdown]...`

<img width="888" alt="Screenshot 2023-02-17 at 15 54 23" src="https://user-images.githubusercontent.com/31280003/219688476-191c5f8d-3c97-4bb3-b4e5-e185bffc7dc5.png">

**Comment by Hornebom** [comment]
@annamehr There are a couple more places that are conditionally shown, that might be rewritten
- The epoch is continuing until orders have been locked and can be executed.
- The epoch continues until all orders can be executed. Close the epoch to partially execute orders and lock the remaining orders into the following epoch.
- The pool currently may be oversubscribed for additional investments or there is insufficient liquidity available for redemptions. Closing of the epoch will not ensure execution of pending orders.

**Comment by annamehr** [comment]
@Hornebom I changed them in the end because it would be confusing to leave epochs reference in there

CURRENT: The epoch is continuing until orders have been locked and can be executed.
NEW:  The collection of orders is continuing until orders have been locked and can be executed.
CURRENT: The epoch continues until all orders can be executed. Close the epoch to partially execute orders and lock the remaining orders into the following epoch.
NEW:  The collection of orders continues until all orders can be executed. Start the execution of the orders to partially execute orders and lock the remaining orders into the next cycle of order executions.
CURRENT: The pool currently may be oversubscribed for additional investments or there is insufficient liquidity available for redemptions. Closing of the epoch will not ensure execution of pending orders.
NEW: The pool currently may be oversubscribed for additional investments or there is insufficient liquidity available for redemptions. Closing of the collection of orders epoch will not ensure execution of pending orders.
